modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-configuring-custom-metrics_{context}"]
+= Configuring custom metrics
+. In the web console, navigate to the Import icon, +. 
+. Here are a couple of examples of `FlowMetrics` configurations you can add:
++
+.Generate a metric that tracks ingress bytes received from cluster external sources
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flowmetric-cluster-external-ingress-traffic
+spec:
+  metricName: cluster_external_ingress_bytes_total  <1>
+  type: Counter                                     <2>
+  valueField: Bytes
+  direction: Ingress                                <3>
+  labels: [DstK8S_HostName,DstK8S_Namespace,DstK8S_OwnerName,DstK8S_OwnerType] <4>
+  filters:                                          <5>
+  - field: SrcSubnetLabel
+    matchType: Absence
+----
+<1> How the metrics appear, along with the prefix `netobserv`. 
+<2> The `type` `Counter` is useful for counting bytes or packets. Histogram is the other available type, which is useful for latencies. 
+<3> Direction of traffic to capture. Egress is an alternative option. If not specified, both ingress and egress are captured. 
```

Comment:
perhaps add something like:
"[...] are captured, which can lead to duplicated counts."

This is so users understand the reason for specifying a direction: it avoids duplication, since a single packet flowing from A to B will be reported once by A and once by B, leading in duplication. Specifying Ingress or Egress tells to report it only once. We could write a whole page explaining the subtelties of reporters & duplication, but trying here to tell just the essentials.. :-)

Reply:
Thanks joel. I agree its good to explain subtleties like this via examples in code callouts than in conceptual pieces.